README.md

```diff
@@ -22,14 +22,27 @@ devtools::install_github("PecanProject/rpecanapi")
 _The following snippets assume that the PEcAn RESTful API Server is running on `http://localhost:8000`. This can be replaced with any
 other appropriate PEcAn Server URL._
 
-### Load `rpecanapi` & Create the Server Object
+- [Getting Started](#getting-started)
+- [Get Information about PEcAn Models](#get-information-about-pecan-models)
+- [Get Information about PEcAn Sites](#get-information-about-pecan-sites)
+- [Get Information about PEcAn PFTs (Plant Functional Types)](#get-information-about-pecan-pfts-plant-functional-types)
+- [Get Information about PEcAn Formats](#get-information-about-pecan-formats)
+- [Get Information about PEcAn Inputs](#get-information-about-pecan-inputs)
+- [Get Information about PEcAn Workflows](#get-information-about-pecan-workflows)
+- [Submit a PEcAn Workflow for Execution](#submit-a-pecan-workflow-for-execution)
+- [Get Information about PEcAn Runs](#get-information-about-pecan-runs)
+- [Plots & File Downloads for Workflows & Runs](#plots--file-downloads-for-workflows--runs)
```

Comment:
There's a TON of info in this README that users will need/want to know. Do we know if this gets built into the package documentation? If not, I wonder if much of this should be moved to a vignette instead? Related, it would be good to add pkgdown support to your package.

Reply:
As of now, I haven't set any way in which this README would be directly available to users in the form of package docs. I intended to discuss this very issue with @robkooper over our weekly call today...but seems he's busy. Will talk to him & get it figured out soon.